health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/downloadTemplate.js

```
@@ -0,0 +1,369 @@
+      
+      function performUnfreezeCells(sheet) {
+        // logger.info(`Unfreezing the sheet ${sheet.name}`);
+      
+        let lastFilledColumn = 1;
+        sheet.getRow(1).eachCell((cell, colNumber) => {
+          if (cell.value !== undefined && cell.value !== null && cell.value !== '') {
+            lastFilledColumn = colNumber;
+          }
+        });
+      
+        for (let row = 1; row <= parseInt("10000"); row++) {
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)